scripts/prepare_pr_comment.py

```diff
@@ -1,8 +1,18 @@
+import os
 import sys
 
 def prepare_failure_commet(repository, issue_number, vendor_label, chart_name):
     msg = f"Thank you for the PR #{issue_number}!\n\n"
-    msg += f"There are [some errors](https://github.com/{repository}/pull/{issue_number}/checks) while building and validating your PR. Please check the log for more details.\n\n"
+    runid = open("./pr/build-verify-check-run-id").read()
+    run_url = f"https://github.com/{repository}/pull/{issue_number}/checks?check_run_id={runid}".strip()
+    msg += f'There are some errors while building and validating your PR.'
+    msg += 'Please check the log for more details:\n'
+    msg += run_url
+    msg += '\n(Please open this link in another tab/window)\n\n'
+    if os.path.exists("./pr/errors"):
+        msg += f"These are the critical issues:\n\n"
+        msg += open("./pr/errors").read()
+    msg += "\n---\n"
```

Comment:
would you mind to switch to multiline string, they read better IMHO. For example:

msg = f"""
   Thank you for the PR #{issue_number}!

   There are some errors while building and validating your PR.

"""

Reply:
Notice mixed use of single quote and double quote, if not need for some python reason consider using one or the other.

Reply:
Done. 562c786b9c7ff2c7caf66d63f731dbff939e43f1